src/frontend/athenaeum/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "athenaeum",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint": "^7.0.0",
```

Comment:
Let's change the version to `6.8.0`, because React requires that version for their internal tooling for some odd reason.